### ISSUE TITLE
[Wiki] Make wiki page about integrating into PEX neutral

### ISSUE THREAD

**Issue by Andre601** [CodeCrafter47/AdvancedTabOverlay#5]
The page [Integrating with PermissionsEx](https://github.com/CodeCrafter47/AdvancedTabOverlay/wiki/PermissionsEx) is a bit in favour of using PEX over any other permission plugin.

While I understand that people out there still use PEX, would I suggest making the wiki page a bit more neutral, so that it doesn't cater to just a single plugin, but rather any perm plugin that uses Vault in a similar way.

The page could either be called `Integrating with a Permission plugin` or `Integrating with Vault` in that case.

I hope you get my point here.

**Comment by CodeCrafter47** [comment]
I totally agree with this. At the time that wiki page has been created (for BungeeTabListPlus), PEX was way more popular than it is now.

The section about sorting players could use some updates while I'm on it. I feel like using the weight is no longer the best option. Especially with LuckPerms that doesn't lead to the desired order in some cases. I think using a custom meta value is the better option.

**Comment by CodeCrafter47** [comment]
I have replaced the Integrating with PermissionEx page with two new pages:
* [Nicknames and Prefixes](https://github.com/CodeCrafter47/AdvancedTabOverlay/wiki/Nicknames-and-Prefixes) and
* [Sorting Players by Rank](https://github.com/CodeCrafter47/AdvancedTabOverlay/wiki/Sorting-Players-by-Rank).

I have also made significant changes to the rest of the wiki, in a few cases taking some inspiration from the changes in your fork of the BTLP wiki. Feel free to have a look at it. I intend to push the same changes to the BTLP wiki soon.
